bitrepository-audit-trail-service/src/main/java/org/bitrepository/audittrails/store/AuditDatabaseExtractor.java

```diff
@@ -126,12 +124,10 @@ public class AuditDatabaseExtractor {
     private final DBConnector dbConnector;
 
     /**
-     * Constructor.
-     *
      * @param model       The model for the restriction for the extraction from the database.
      * @param dbConnector The connector to the database, where the audit trails are to be extracted.
      */
-    public AuditDatabaseExtractor(ExtractModel model, DBConnector dbConnector) {
```

Comment:
`ExtractModel` isn’t visible outside the package, so no one could benefit from this constructor being public.